internal/controller/testworkflows/testworkflow_controller.go

```diff
@@ -215,3 +223,25 @@ func (r *TestWorkflowReconciler) SetupWithManager(mgr ctrl.Manager) error {
 		For(&testworkflowsv1.TestWorkflow{}).
 		Complete(r)
 }
+
+func (r *TestWorkflowReconciler) deleteTestWorkflow(testWorkflowName, namespace string) (out string, err error) {
+	request, err := http.NewRequest(http.MethodDelete,
```

Comment:
Is that expected behavior? That means that if the Test Workflow is deleted and re-created immediately the executions are deleted immediately

Reply:
The idea is to have similar behaivour when you delete Test workflow using api and using CRD. So when we delete it using api then we also delete executions, this change allows to delete executions in case of CRD deletion as well. This way we will keep system consistence regardless of the Test Workflow deletion method. If the user decides delete/recreate CRD, rather than update, then he/she is responsible for this

Reply:
We can add Purge Executions var to Operator helm chart to support this option . It can be false by default